docs/org.eclipse.n4js.design/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
a `.gitignore` file should be added, similar to the one in the spec plugin

Reply:
added